dmoj/graders/interactive.py

```diff
@@ -0,0 +1,40 @@
+from dmoj.graders.standard import StandardGrader
+
+class Interactor(object):
```

Comment:
This class should also implement something among the lines of `readint`, `readtoken` etc., and catch `ValueError`s as necessary, rethrowing them as a new `WrongAnswer` exception (see below comments).